codecs/oxipng/Cargo.toml

```diff
@@ -12,7 +12,19 @@ crate-type = ["cdylib"]
 oxipng = { version = "3.0.0", default-features = false }
 wasm-bindgen = "0.2.64"
 log = { version = "0.4", features = ["release_max_level_off"] }
+rayon = { version = "1.3.0", optional = true }
+once_cell = { version = "1.3.1", optional = true }
+crossbeam-channel = { version = "0.4.2", optional = true }
+
+[patch.crates-io]
+oxipng = { git = "https://github.com/RReverser/oxipng", branch = "crossbeam" }
 
 [profile.release]
 lto = true
 opt-level = "s"
+
+[features]
+parallel = ["oxipng/parallel", "rayon", "crossbeam-channel", "once_cell"]
+
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = ["-O", "--no-validation"]
```

Comment:
Why `--no-validation`? Isn’t `--enable-bulk-memory --enable-threads` doing the trick?

Reply:
Something kept failing for me, but perhaps I didn't try `--enable-bulk-memory`. Personally, I don't think validation is that important when Wasm is produced by a legitimate compiler, it's mostly useful for cases when you try to build it by hand.

Reply:
Alright.